spec.html

```diff
@@ -38231,7 +38065,7 @@ <h1>
         </dl>
         <emu-alg>
           1. If IsCallable(_adder_) is *false*, throw a *TypeError* exception.
-          1. Assert: _iterable_ is present, and is neither *undefined* nor *null*.
+          1. Assert: _iterable_ is neither *undefined* nor *null*.
```

Comment:
Any reason not to just declare its type to be `an ECMAScript language value, but not *undefined* or *null*`?

Reply:
I wasn't sure if that assertion was conditional upon the IsCallable check above it.

Reply:
It is not.